docs/opensearch-sink-connector-config-options.rst

```diff
@@ -1,44 +1,25 @@
 =========================================
-OpenSearch Sink Connector Configuration Options
+Opensearch Sink Connector Configuration Options
```

Comment:
Please change back to `OpenSearch`, we are constantly being "beaten" by that (see please https://github.com/opensearch-project/OpenSearch/blob/main/README.md)